src/App.js

```diff
@@ -29,37 +29,94 @@ const App = () => {
   // This starts state off as a 2D array of JS objects with
   // empty value and unique ids.
   const [squares, setSquares] = useState(generateSquares());
-
+  // const [winner, setWinner] = useState('');
+  const [currentPlayer, setCurrentPlayer] = useState(Player1);
   // Wave 2
   // You will need to create a method to change the square
   //   When it is clicked on.
   //   Then pass it into the squares as a callback
 
+  const onClickSquare = (id) => {
+    console.log(id);
+    const newSquares = [...squares];
+    if (checkForWinner() === null) {
+      for (let i = 0; i < 3; i++) {
+        for (let j = 0; j < 3; j++) {
+          if (id === newSquares[i][j].id && newSquares[i][j].value === '') {
+            newSquares[i][j].value = currentPlayer;
+          }
+        }
+      }
+    }
+    // setWinner(checkForWinner());
+    setSquares(newSquares);
+    // console.log(winner);
+
+    if (currentPlayer === Player1) {
+      setCurrentPlayer(Player2);
+    } else {
+      setCurrentPlayer(Player1);
+    }
```

Comment:
We can use a ternary operator to set the currentPlayer:

```suggestion
      setCurrentPlayer(currentPlayer === PLAYER_1 ? PLAYER_2 : PLAYER_1;

```